CODEOWNERS

```diff
@@ -1 +1 @@
-* @actions/actions-runtime
+* @actions/actions-runtime
```

Comment:
Please remove. This is inherited from the original source in `@actions` org.

Btw, this should go in `.github` folder whenever it exists